src/troubleshooting/espflash.md

```diff
@@ -0,0 +1,32 @@
+# espflash
+
+## Error when trying to use espflash: `rtc_clk_init: Possibly invalid CONFIG_XTAL_FREQ setting`
```

Comment:
```suggestion
## `rtc_clk_init: Possibly invalid CONFIG_XTAL_FREQ setting`
```

Since we are in the `espflash` section. "Error when trying to use espflash:" does not provide much information